staging_service/metadata.py

```diff
@@ -32,7 +33,11 @@
         data = {}
     # first ouptut of md5sum is the checksum
     data['source'] = source
-    md5 = await run_command('md5sum', path.full_path)
+    try:
+        md5 = hashlib.md5(open(path.full_path, 'rb').read()).hexdigest()
```

Comment:
Is this faster than md5sum?  It certainly is cleaner.

Reply:
@scanon I think performance-wise, it will be likely the same. But it won't throw a 404 error and prevent later executions.

Reply:
I also personally don't like using system command because we will likely have less control.